packages/insomnia-app/app/ui/components/settings/plugins.js

```diff
@@ -20,7 +20,8 @@ import { showAlert, showPrompt } from '../modals';
 type State = {
   plugins: Array<Plugin>,
   npmPluginValue: string,
-  error: string,
+  error: Object,
```

Comment:
This can be typed explicitly :)

```suggestion
  error: Error | null,
```